### PR TITLE
old argument targets->labels softmax_cross_entropy_with_log

### PR DESCRIPTION
**Pull request by Asiapenolove**
for the tf 1.0 usage:
tf.nn.softmax_cross_entropy_with_logits(_sentinel=None, labels=None, logits=None, dim=-1, name=None)
update targets to labels

https://www.tensorflow.org/api_docs/python/tf/nn/softmax_cross_entropy_with_logits

### CODE REVIEW

**Comment by Asiapenolove** [comment]
I signed it!

**Comment by nealwu** [comment]
Thanks!
